src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -254,17 +256,16 @@ private function transition($subject, Transition $transition, Marking $marking)
 
     private function enter($subject, Transition $transition, Marking $marking)
     {
+        foreach ($enter = $transition->getTos() as $place) {
```

Comment:
`enter` -> `places` ? because right now, it has the same name as the method. It's weird to me. And So update the `$from` from the `leave` method to be consistent.